homeassistant/components/apsystems/switch.py

```diff
@@ -0,0 +1,67 @@
+"""The power switch which can be toggled via the APsystems local API integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from aiohttp.client_exceptions import ClientConnectionError
+from APsystemsEZ1 import Status
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import DiscoveryInfoType
+
+from . import ApSystemsConfigEntry, ApSystemsData
+from .entity import ApSystemsEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ApSystemsConfigEntry,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the switch platform."""
+
+    add_entities([ApSystemsPowerSwitch(config_entry.runtime_data)])
+
+
+class ApSystemsPowerSwitch(ApSystemsEntity, SwitchEntity):
+    """Base switch to be used with description."""
+
+    _attr_device_class = SwitchDeviceClass.SWITCH
+
+    def __init__(
+        self,
+        data: ApSystemsData,
+    ) -> None:
```

Comment:
```suggestion
    def __init__(self, data: ApSystemsData) -> None:
```

Reply:
I think it's short enough to be on one line, but the comma at the end makes ruff go to the second line.

But it's also possible that ruff adds the comma, in that case this can be ignored